src/main/java/BasisStatus/StatusIdle.java

```diff
@@ -905,7 +905,7 @@ public void mouseMoved(MouseEvent e) {
 
             try {
                 vmObject.processAddElement(mainPath, circuitClass, frontClass, args);
-            } catch (Exception ex) {
+            } /* TODO why the exception here? */ catch (Exception ex) {
```

Comment:
Maybe it calls functions on elements. Because users can add own element implementations (I think) we need to make sure no exception is thrown. But this assumption probably has too much faith in whoever write this.